icons/geode.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Geode</title><path d="M12 0a1.41 1.41 0 0 0-.8574.289L.5957 8.3439a1.412 1.412 0 0 0-.4922 1.539L4.1602 23.004a1.4135 1.4135 0 0 0 1.3496.9961h12.9804a1.4135 1.4135 0 0 0 1.3496-.996l4.0567-13.1212a1.412 1.412 0 0 0-.4922-1.539l-.9727-.7422-1.1191 3.5175-3.0703 9.9903a.929.929 0 0 1-.8887.6562H6.5781a.929.929 0 0 1-.8886-.6562L2.328 10.1758a.931.931 0 0 1 .3223-1.0117l8.748-6.7168a.9.9 0 0 1 .1094-.0723l2.045-1.5547-.6954-.5312A1.41 1.41 0 0 0 12 0m.0684 4.7578a.756.756 0 0 0-.461.1563L5.084 9.918a.758.758 0 0 0-.2637.8242l.4336 1.4082 2.1973-1.6191 4.3144-3.377a.3816.3816 0 0 1 .4688 0l4.705 3.6816a.376.376 0 0 1 .129.4063l-1.8086 5.9922a.38.38 0 0 1-.3633.2695h-5.793a.4.4 0 0 1-.0762-.0078h-2.127l.4278 1.3926a.7585.7585 0 0 0 .7246.5351h8.0332a.756.756 0 0 0 .7227-.5351l2.5098-8.1465a.758.758 0 0 0-.2637-.8242l-6.5254-5.004a.756.756 0 0 0-.461-.1562"/></svg>
```

Comment:
Which logo did you use from the Zip file?
I compared with `inner_logo_no_shadow.svg` and I'm seeing some differences.

Reply:
I believe thats what I used as well... I'll check again!

Reply:
I've updated the icon, please check again!